### PR TITLE
Added legacy SBOM deprecation notice.

### PR DESCRIPTION
**Pull request by ncarlson**
https://github.com/pivotal/build-service/issues/870

### CODE REVIEW

**Review by tomkennedy513 — APPROVED** [review]

File: getting-started.html.md.erb [hunk 1]
```diff
@@ -52,6 +52,14 @@ For this example setup, we make the following assumptions:
   * The nodes on your cluster must also be configured to trust this CA certificate so they can pull in images. Configuration for this depends on the cluster provider
 * You are using an "online" environment that has access to the internet
 
+## <a id='DeprecatedFeatures'></a> Deprecated Features
+
+* Users are encouraged to use the SBOM format published in
+[Buildpacks API 0.7.0](https://buildpacks.io/docs/reference/spec/migration/buildpack-api-0.6-0.7/#new-standardized-sbom-format).
+Older (legacy) SBOM formats are deprecated and are disabled by default in TBS. Support for legacy SBOM formats be
```
Comment (tylerphelan) [inline getting-started.html.md.erb:59]: maybe remove s from the legacy bom as the "s" implies the new bom.

`Older (legacy) SBOM formats are deprecated and are disabled by default in TBS. Support for legacy SBOM formats be` -> `The (legacy) CNB BOM format is deprecated and disabled by default in TBS. Support for legacy BOM can be`

File: installing-no-kapp.html.md.erb [hunk 2]
```diff
@@ -59,6 +59,14 @@ for authentication with registries.
 
 <p class="note"><strong>Note:</strong> TKGs clusters running Kubernetes 1.20.0-1.20.6 are not compatible with Tanzu Build Service.</p>
 
+## <a id='DeprecatedFeatures'></a> Deprecated Features
+
+* Users are encouraged to use the SBOM format published in
+[Buildpacks API 0.7.0](https://buildpacks.io/docs/reference/spec/migration/buildpack-api-0.6-0.7/#new-standardized-sbom-format).
+Older (legacy) SBOM formats are deprecated and are disabled by default in TBS. Support for legacy SBOM formats be
```
Comment (tomkennedy513) [inline installing-no-kapp.html.md.erb:66]: Enabled by default

**Review by tomkennedy513 — CHANGES_REQUESTED** [review]

File: getting-started.html.md.erb [hunk 3]
```diff
@@ -52,6 +52,14 @@ For this example setup, we make the following assumptions:
   * The nodes on your cluster must also be configured to trust this CA certificate so they can pull in images. Configuration for this depends on the cluster provider
 * You are using an "online" environment that has access to the internet
 
+## <a id='DeprecatedFeatures'></a> Deprecated Features
+
+* Users are encouraged to use the SBOM format published in
+[Buildpacks API 0.7.0](https://buildpacks.io/docs/reference/spec/migration/buildpack-api-0.6-0.7/#new-standardized-sbom-format).
```
Comment (matthewmcnew) [inline getting-started.html.md.erb:58]: The buildpacks api is really intended for buildpack authors not users of TBS. Perhaps, linking to https://buildpacks.io/docs/tools/pack/cli/pack_sbom_download/ would be more appropriate?

Comment (ncarlson) [reply getting-started.html.md.erb]: I've removed the mention of buildpacks, as it does seem out of place.

File: managing-images.html.md.erb [hunk 4]
```diff
@@ -785,6 +785,10 @@ Build successful
 ```
 ## <a id='build-status'></a> Viewing bill of materials
 
+> **Deprecation Note:**
+> The (legacy) CNB BOM format is deprecated, but enabled by default in TBS. Support can be disabled manually by passing
+> `--data-value-yaml include_legacy_bom=false` to `ytt` when processing the TBS bundle.
+
 The kp cli allows you to view the bill of materials in an image built by a Build.
```
Comment (tylerphelan) [inline managing-images.html.md.erb:792]: >The kp cli allows you to view the bill of materials in an image built by a Build.
we should mention this is no longer supported in kp 0.7.0+

**Review by tylerphelan — APPROVED** [review]

File: getting-started.html.md.erb (hunk 1 above)
Comment (ncarlson) [reply getting-started.html.md.erb]: Thanks, @tylerphelan. I reverted this PR and opened a new draft PR with your suggestions. Found here: https://github.com/pivotal/docs-tap/pull/1570

File: installing-no-kapp.html.md.erb (hunk 2 above)
Comment (ncarlson) [reply installing-no-kapp.html.md.erb]: Thanks, @tomkennedy513.  I reverted this PR and opened a new draft PR with your suggestions. Found here: https://github.com/pivotal/docs-tap/pull/1570

File: getting-started.html.md.erb (hunk 3 above)
Comment (ncarlson) [reply getting-started.html.md.erb]: Thanks, @matthewmcnew. I reverted this PR and opened a new draft PR here: https://github.com/pivotal/docs-tap/pull/1570.

File: managing-images.html.md.erb (hunk 4 above)
Comment (ncarlson) [reply managing-images.html.md.erb:796]: Thank you. Added it.

**Pull request merged**